qt/python/mantidqt/mantidqt/widgets/workspacedisplay/table/model.py

```diff
@@ -28,6 +29,13 @@ class TableWorkspaceColumnTypeMapping(object):
     LABEL = 6
 
 
+@contextmanager
+def table_ws_context_manager(model):
+    model.edit_flag = True
```

Comment:
Slightly nit picky but could we call this flag something like `self.block_model_replace`. I think then we want to invert the logic here as we want to disable the replacement temporarily?

Reply:
I think the logic is fine. It would start and set `block_model_replace` to True so we will "block signals". This is the same as "it is being edited" which is what I was thinking of when I wrote it.